inotify-tools/plan.sh

```diff
@@ -0,0 +1,12 @@
+pkg_origin=core
+pkg_name=inotify-tools
+pkg_version=3.14
+pkg_description="inotify-tools is a C library and a set of command-line programs for Linux providing a simple interface to inotify. These programs can be used to monitor and act upon filesystem events."
+pkg_upstream_url=https://github.com/rvoicilas/inotify-tools/wiki
+pkg_license=("GNU-2.0")
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_source="http://github.com/downloads/rvoicilas/${pkg_name}/${pkg_name}-${pkg_version}.tar.gz"
+pkg_shasum=222bcca8893d7bf8a1ce207fb39ceead5233b5015623d099392e95197676c92f
+pkg_deps=()
```

Comment:
I think it is probably a good idea to add `core/glibc` to this list:

```
[15][default:/src:0]# ldd /hab/pkgs/core/inotify-tools/3.14/20171204103835/bin/inotifywait
        linux-vdso.so.1 (0x00007ffceafd6000)
        libinotifytools.so.0 => /hab/pkgs/core/inotify-tools/3.14/20171204103835/lib/libinotifytools.so.0 (0x00007f078a036000)
        libc.so.6 => /hab/pkgs/core/glibc/2.22/20170513201042/lib/libc.so.6 (0x00007f0789c92000)
        /hab/pkgs/core/glibc/2.22/20170513201042/lib/ld-linux-x86-64.so.2 (0x00007f078a252000)
```

While it works out with explicitly adding the dependency because of how gcc is put together, listing it as a dependency will ensure this gets rebuild when glibc gets bumped.